cftime/_cftime.pyx

```diff
@@ -304,6 +297,62 @@ def num2date(times,units,calendar='standard',\
     returns a datetime instance, or an array of datetime instances with
     approximately 100 microsecond accuracy.
 
+    ***Note***: If only_use_cftime_datetimes=False and
+    use_only_python_datetimes=False, the datetime instances
+    returned are 'real' python datetime
+    objects if `calendar='proleptic_gregorian'`, or
+    `calendar='standard'` or `'gregorian'`
+    and the date is after the breakpoint between the Julian and
+    Gregorian calendars (1582-10-15). Otherwise, they are ctime.datetime
+    objects which support some but not all the methods of native python
+    datetime objects. The datetime instances
+    do not contain a time-zone offset, even if the specified `units`
+    contains one.
+    """
+    return num2date_float(
```

Comment:
Let's go ahead, bite the bullet,  and use `numdate_int` as long as all the tests pass.

Reply:
Sure thing; it looks like they all do now.